src/checkout/views/Review/View.tsx

```diff
@@ -0,0 +1,120 @@
+import "./scss/index.scss";
+
+import { History } from "history";
+import * as React from "react";
+
+import { Redirect, RouteComponentProps } from "react-router";
+
+import {
+  Button,
+  OverlayContext,
+  OverlayType,
+  ShowOverlayType
+} from "../../../components";
+import { BASE_URL } from "../../../core/config";
+import { AddressSummary } from "../../components";
+import { CheckoutContext } from "../../context";
+import { baseUrl as checkoutBaseUrl } from "../../routes";
+import CartTable from "./CartTable";
+import { TypedCompleteCheckoutMutation } from "./queries";
+import { completeCheckout } from "./types/completeCheckout";
+
+const completeCheckout = (
+  data: completeCheckout,
+  show: ShowOverlayType,
+  history: History,
+  clear: () => void
+) => {
+  const canProceed = !data.checkoutComplete.errors.length;
+
+  if (canProceed) {
+    history.push(BASE_URL);
```

Comment:
Can we use just `<Redirect />` component to maintain consistency in project?

Reply:
I'll improve this page in #222